src/test/scala/uk/gov/nationalarchives/consignmentexport/BagMetadataSpec.scala

```diff
@@ -34,6 +35,7 @@ class BagMetadataSpec extends ExportSpec {
     bagMetadata.get("Consignment-CompletedDate").get(0) should be(fixedDateTime.toFormattedPrecisionString)
     bagMetadata.get("Contact-Name").get(0) should be("FirstName LastName")
     bagMetadata.get("Consignment-ExportDate").get(0) should be(fixedDateTime.toFormattedPrecisionString)
+    bagMetadata.get("Bag-Creator").get(0) should be(version)
```

Comment:
Now I see this in context, it looks a bit odd. Was Phil expecting just a version number in this field, or should we include the name too? e.g. `tdr-consignment-export-v1.2.3` rather than just `1.2.3`?

Reply:
It would probably be better with a bit of context. I can prepend the version with `tdr-consigment-export-v` in the `BagMetadata` class.

Reply:
Can you check with Phil? I was just thinking out loud - I don't know if there's a good reason to just use the version number.

Reply:
I've spoken to him, he's asked for `TDRExportv0.0.1` so I've changed it in the class and in the test.